### DISCUSSION TITLE
inputNumber doesn't accept entering ".5"

### DISCUSSION THREAD

**Discussion by iMarti** [primefaces/community#525] [PrimeNG]
Entering a number starting with a dot, like ".5" instead of "0.5" is not possible. 
Try it in the third "Min-Max Fraction Digits" box on https://primeng.org/inputnumber
This is a common need actually supported by the standard `<input type="number">` element.

Is there a workaround or is this a bug?

**Comment by cetincakiroglu** [comment]
Hi,

Mentioned feature is not supported by inputNumber component. It's a good idea to make a support for it. Thanks for reporting the issue we've added it in our internal backlog.

**Reply by abhijit-chikane** [reply]
I was wondering if there's been any progress on this?
